ocdsdata/cli/commands/run.py

```diff
@@ -39,18 +38,24 @@ def configure_subparser(self, subparser):
     def run_command(self, args):
         run = []
 
-        if args.runall:
+        if args.all and args.source:
+            print("You need to either specifiy a source or use all flag, not both.")
+            quit(-1)
+
+        if args.all:
             for source_id, source_class in self.sources.items():
                 run.append({'id': source_id, 'source_class': source_class})
-        elif args.run:
-            if args.run in self.sources:
-                run.append({'id': args.run, 'source_class': self.sources[args.run]})
-            else:
-                print("We can not find a source that you requested! You requested: %s" % [args.run])
-                quit(-1)
+        elif args.source:
+            for selected_source in args.source:
+                print(selected_source)
```

Comment:
I imagine this print statement should be removed. (We aren't printing sources when `--all` is set).